### ISSUE TITLE
Member Ordering with constructor

### ISSUE THREAD

**Issue by trustylogic** [palantir/tslint#1251]
### Bug Report

- __TSLint version__: `3.10.1`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__: (please choose one) Webpack

#### TypeScript code being linted

This is just a watered down example.

```ts
	export function FactoryGet(meh) {
		return class Res {
			constructor(res: string) {
				return res;
			}
		};
	}
```

with `tslint.json`:

```json
    "member-ordering": [true,
        "static-before-instance"
    ],
```

#### Actual behavior

I'm getting this error:

......./node_modules/tslint/lib/rules/memberOrderingRule.js:212
        this.memberStack[this.memberStack.length - 1].push(node);
TypeError: Cannot read property 'push' of undefined

#### Expected behavior

Not sure why i am getting this error... I'm fairly new to Typescript and Tslint.

I think it has something to do with the contractor call...

Any help would be appreciated. Thanks.

**Comment by adidahiya** [comment]
related to #1243

**Comment by JKillian** [comment]
Thanks for the report! This can be fixed with the same fix as #1250

**Comment by trustylogic** [comment]
Thanks... is there going to be a new release with this update coming soon?
